massa-api/src/public.rs

```diff
@@ -404,7 +404,11 @@ impl MassaRpcServer for API<Public> {
         Ok(consensus_controller.get_cliques())
     }
 
-    async fn get_stakers(&self) -> RpcResult<Vec<(Address, u64)>> {
+    async fn get_stakers(
+        &self,
+        limit: Option<usize>,
```

Comment:
It'll be better to create a struct PageRequest

could be inspired by https://docs.spring.io/spring-data/data-commons/docs/current/api/org/springframework/data/domain/PageRequest.html